src/api-reference/financial-integration/v4.financial-integration.markdown

```diff
@@ -429,24 +607,44 @@ Code|Description|Category
 
 Name|Type|Format|Description
 ---|---|---|---
-`fIDocuments`|`Array`|[Financial document](#schema-fidocument)|The result collection.
-`nextPage`|`string`|-|The URI of the next page of results.
-`numberReturned`|`int32`|-|Number of documents returned in current page.
-`offset`|`string`|-|Next page offset.
+`content`|`Array`|[Financial document](#schema-fidocument)|The result collection.
```

Comment:
This link text should be FIDocument, if swagger is correct.

Reply:
done, please check